templates_for_code_gen/filter_folder/Cargo.toml.handlebars

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
This looks like an odd folder structure, why are the filter templates nested like that?

Reply:
If that is a new additional back end, it should probably be in the src/sim_backend folder.

Reply:
It's not a new backend;  it's just the templates used to create a compiled rust library.  I thought it improved readability to keep the files nested in the same way that they are in the compiled library.  That way you can more easily see how the files are meant to interact, eg, the functions in graph_utils.rs are used in filter.rs.

Reply:
I guess the other comment I made applies here. It looks like this has the same functionality as the `cpp_filter` dir, so we should probably use it the same way.